templates/patterns/timeline/timeline.ui_patterns.yml

```diff
@@ -0,0 +1,58 @@
+timeline:
```

Comment:
This file should be the same as https://github.com/openeuropa/oe_theme/blob/master/templates/patterns/timeline/timeline.ui_patterns.yml

Reply:
Ok we could cut new functionality of ECL2 where we could specify from what element show again again lines.
But what we have to do with `button label` (show again hidden items). Because in ecl2 we have to show something?
It this case it should be configurable anymore?
Also question about structure of `items` elements.

Reply:
I know that it's painful, but we should take into consideration the legacy of this timeline pattern.
We don't cut any functionality of ECL2, we should adapt our code.
So for this case, we could add an extra field:
```
    button_label_hidden:
      type: "text"
      label: "Button label hidden"
      description: "Hidden timeline button label."
      preview: "Hidden all timeline items"
```

For item it's the same, the pattern should be more generic and use the preprocess and the template to transform for ecl2.

Reply:
Ok. Thank you for input. I have taken in consideration and finished rework.